authorization/invitation/service/invitation_service.go

```diff
@@ -333,6 +334,25 @@ func lookupSpaceName(ctx context.Context, witURL string, spaceID string) (string
 	return *spaceSingle.Data.Attributes.Name, nil
 }
 
+// Rescind revokes an invitation request
+func (s *invitationServiceImpl) Rescind(ctx context.Context, issuingUserID, invitationID uuid.UUID) error {
+
+	err := s.ExecuteInTransaction(func() error {
```

Comment:
There should be a permission check here, we need to confirm that the current user has the rights to rescind the invitation. The type of permission check you need to perform depends on whether the `inviteTo` or `resourceID` property is set in the invitation.  If `inviteTo` is set, you need to lookup the identity with that ID, then do the permission check on its resource (`IdentityResourceID`). If `resourceID` is set, you do the permission check on that:

```
// Create the permission service
permService := s.Services().PermissionService()

// Confirm that the issuing user has the necessary scope to manage members for the resource
err := permService.RequireScope(ctx, issuingUserId, <resource ID to check>, authorization.ScopeForManagingRolesInResourceType(<resource type name>)
if err != nil {
    return err
}
```